src/tests/changelist.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { INestApplication } from '@nestjs/common'
+import * as request from 'supertest'
+import { TypeOrmModule } from '@nestjs/typeorm'
+import { UserModule } from '../../exampleApp/src/user/user.module'
+import { TestAuthModule } from '../../exampleApp/test/testAuth/testAuth.module'
+import { AdminCoreModuleFactory } from '../adminCore.module'
+import { JSDOM } from 'jsdom'
+
+describe('changelist', () => {
+  let app: INestApplication
+  let document: Document
+
+  beforeAll(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        TypeOrmModule.forRoot(),
+        UserModule,
+        TestAuthModule,
+        AdminCoreModuleFactory.createAdminCoreModule({}),
+      ],
+    }).compile()
+
+    app = module.createNestApplication()
+    await app.init()
+  })
+
+  afterAll(async () => {
+    await app.close()
+  })
+
+  beforeEach(() => {
+    const dom = new JSDOM()
+    document = dom.window.document
+  })
+
+  it('can show the columns defined in listDisplay', async () => {
+    const server = app.getHttpServer()
+    const res = await request(server).get(`/admin/user/user`)
+
+    expect(res.status).toBe(200)
+
+    document.documentElement.innerHTML = res.text
+    expect(
+      document.querySelector('table > thead > tr > th:nth-child(1)').innerHTML.includes('User'),
```

Comment:
```suggestion
      document.querySelector('table th:nth-child(1)').innerHTML.includes('User'),
```

Same for below. No point making the selector too specific, it makes the test more brittle